src/api/logger.js

```diff
@@ -0,0 +1,57 @@
+import winston from "winston";
+
+// Define the severity levels
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+};
+
+/*
+ If the server was on development mode, show all the log levels.
+ If the server was on production mode, show only warn and error messages.
+*/
+const level = () => {
+  const env = process.env.NODE_ENV || 'development'
+  const isDevelopment = env === 'development'
+  return isDevelopment ? 'debug' : 'warn'
+};
+
+const colors = {
+  error: 'red',
+  warn: 'yellow',
+  info: 'green',
+  http: 'magenta',
+  debug: 'white',
+};
+
+// Show different color for different severity levels.
+winston.addColors(colors);
+
+const format = winston.format.combine(
+  // Add the message timestamp
+  winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss:ms' }),
```

Comment:
I usually see the log timestamp format to be in ISO8601, i.e. https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations. Is it something that can be done as it makes thing unambiguous.

Another question: 
Does the logger use UTC timezone for capturing time? If it does not, I would recommend to use it. It removes the ambiguity from the time.